internal/serialization/class_definition_context.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
```

Comment:
Just a little comment, the year should be 2023.

Reply:
The copyright year is not very important.